### DISCUSSION TITLE
V2.2.4-Docker build from Source failing with error "sed: can't read /etc/sudoers: No such file or directory"

### DISCUSSION THREAD

**Discussion by Vishal2696** [apache/airflow#22335] [Q&A]
I'm trying to build the latest version of airflow (2.2.4) from source code (https://github.com/apache/airflow/) but unable to do due to an error when I build using the provided dockerfile. Below is the error message. 

```
Step 104/116 : RUN sed --in-place=.bak "s/secure_path=\"/secure_path=\"\/.venv\/bin:/" /etc/sudoers
 ---> Running in 52e85d26079f
sed: can't read /etc/sudoers: No such file or directory

```

Why is there no `/etc/sudoers` file in the container if it is required? I can't manually edit the dockerfile to create this path as this process is part of a CI pipeline using Azure that builds from the repo directly on the flow.

**Comment by potiuk** [comment]
Which version you use (branch/commit sha?) do you use and what is the command you use? The Docker image itself is built on CI of Airflow so it builds in it's basic form for sure, but the question is what is your build command you use?

**Reply by Vishal2696** [reply]
Commit used for build: ee9049c0566b2539a247687de05f9cffa008f871
 Build Command Used: `docker build -t <myimagename>:<mytag>`

**Reply by potiuk** [reply]
Can't reproduce. Built it using the same commit and I have no problems whatsoever. My best guess is that you had some temporary problem with resources/docker engine when you tried to build it. I had to change AIRFLOW_VERSION to released one (2.2.4)

`docker buikd . --build-arg AIRFLOW_VERSION=2.2.4` but other than that it worked just fine.
